docs/examples.rst

```diff
@@ -0,0 +1,183 @@
+    Flow Production Tracking. That means if someone modified worklog in Jira when the Task synced was disabled,
+    and this is worklog already has an associated PTR Timelog, the changes won't be reflected in Flow Production
+    Tracking.
+
+.. note::
+    Timelog and Worklog deletions are not taken into account when doing a full Task synchronization.
+
+
+Configuration
+=============
+
+Setting up Flow Production Tracking
+-----------------------------------
+
+The following field must be created in Flow Production Tracking for each of the
+TimeLog entity type:
+
+===========  =========  ================  ====================================  ======================
+Entity Type  Data Type  Display Name      Description                           Field Name (auto-generated)
+===========  =========  ================  ====================================  ======================
+TimeLog      Text       Jira Key          Synced Issue Key value in Jira        ``sg_jira_key``
+===========  =========  ================  ====================================  ======================
+
+.. note::
+    Make sure the field is configured with the "*Ensure unique
+    value per project*" setting **checked**. This setting can be found by
+    showing the relevant field in an entity spreadsheet view and then
+    right clicking the header for that column. Select the ``Configure field...``
+    menu option.
+
+Setting up Jira
+---------------
+
+Because of the Jira API limitations, it is not possible to set the Jira Worklog assignee.
+To keep a track of the Flow Production Tracking Timelog assignees, we are using a custom field
+on the Jira Issue entity that need to be created.
+
++------------------+-----------+--------------------------------------------------------------+
+| Field Name       | Type      | Description                                                  |
++==================+===========+==============================================================+
+| Shotgun TimeLogs | TextField | Stores the Flow Production Tracking Timelog assignees        |
++------------------+-----------+--------------------------------------------------------------+
+
+.. note::
+    Because this custom field will store JSON data, we can't use a ``Text`` field as it has
+    a character length limitation. This is why we are using a ``TextField`` field.
+    Unfortunately, this type of field doesn't have a ``read-only`` property.
+
+Jira API Limitations & Workaround
+=================================
+
+Because of some Jira API limitations, it is not possible right now to set or update the Jira Worklog assignees.
+In order to keep a track of the Flow Production Tracking Timelog assignees, we have implemented a generic solution
+using a custom field on the Issue entity.
+
+If you want to implement your own solution, it is possible to modify the content of the following methods in
+the Timelog syncer:
```

Comment:
Should we mention that some other third party Jira plugins do allow setting the assignee via their API?
I guess we have not tested this, although we may get chance to try with a customer.

Reply:
I'd prefer not for the moment
As you pointed out, we haven't tested this yet, so I don't want to mention them unless I'm sure at 100% that they're working in this context.
If, at some point, we're going to use them for a customer, I guess we could update the doc and add them